assets/css/v2/pre_fare/body_right/surge.scss

```diff
@@ -0,0 +1,25 @@
+// body total height: 1648px
+
+.body-right-surge {
+  position: relative;
+}
+
+.body-right-surge__upper {
+  position: absolute;
+  top: 0;
+  left: 0;
+
+  width: 1080px;
+  height: 876px;
+  overflow: hidden;
+}
+
+.body-right-surge__lower {
+  position: absolute;
+  top: 918px;
```

Comment:
Should this be `876px`? I think normally we put one slight immediately next to (or below) the previous one, so I would have expected this `top:` to have the same value as the `height:` of the upper slot.

Reply:
Thanks for bumping this. I started to add some numbers initially until I realized we didn't know dimensions yet. Now that we have them, I'll make these match the final numbers in the designs.